### DISCUSSION TITLE
Regd. Expiry of SAS token of Azure Storage Account

### DISCUSSION THREAD

**Discussion by monojit18** [Sunbird-Ed/Community#205] [Installation] [answered]
Dear Team,

Branch: release-4.10.1_RC1

In case of the expiry of Azure storage SAS Token:

Can the already deployed application or any of its modules be affected?
If I checkin with ansible files with the new SaSToken, which of the jobs (Build and Deploy) I have to run again?

Thanks in advance!

**Comment by beepdot** [comment, marked as answer]
@saiakhil46 @santhosh-tg @G33tha  Can you please let know the dependencies on sas token? If possible, please provide the details / process on how to check for the dependencies based on variable so the user can find it themselves instead of raising discussion threads.

**Reply by monojit18** [reply]
This is resolved now and understood; marking it as answered.
